src/power_grid_model/utils.py

```diff
@@ -14,6 +14,8 @@
 
 import numpy as np
 
+from power_grid_model._utils import get_and_verify_batch_sizes as _get_and_verify_batch_sizes
+from power_grid_model._utils import get_batch_size as _get_batch_size
```

Comment:
ahhh i know what this is. @nitbharambe also found this. The solution is to do this change in the `pyproject.toml`:

https://github.com/Alliander/power-grid-model-conversions/pull/104/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R62

please do that change instead of trying to find against a sucky `isort` default setting